libs/native-federation/src/utils/i18n.ts

```diff
@@ -76,7 +76,9 @@ export async function translateFederationArtefacts(
 
   const sourceLocalePath = path.join(outputPath, 'browser', sourceLocale);
 
-  const cmd = `node node_modules/.bin/localize-translate -r ${sourceLocalePath} -s "${sourcePattern}" -t ${translationFiles} -o ${translationOutPath} --target-locales ${targetLocales} -l ${sourceLocale}`;
+  const localizeTranslate = path.resolve("node_modules/.bin/localize-translate");
+
+  const cmd = `${localizeTranslate} -r ${sourceLocalePath} -s "${sourcePattern}" -t ${translationFiles} -o ${translationOutPath} --target-locales ${targetLocales} -l ${sourceLocale}`;
```

Comment:
also just removing "node" in the cmd makes some trouble so i use `path.resolve()`.